src/Maestro/tests/Maestro.ScenarioTests/TestParameters.cs

```diff
@@ -92,6 +97,8 @@ private TestParameters(string darcExePath, string gitExePath, string maestroBase
 
         public GitHubClient GitHubApi { get; }
 
+        public Microsoft.DotNet.DarcLib.AzureDevOpsClient AzDoClient { get; }
```

Comment:
Is there a conflict if you add a `using Microsoft.DotNet.DarcLib`?

Reply:
Unfortunately yeah, there are a lot of things that overlap with OctoKit and I'm using the DarcLib stuff in fewer places in the file so that's the one that makes sense (even though the namespace is longer).